subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/rmf/RMFPProDaysSinceSubscribedMatchingAttribute.kt

```diff
@@ -76,13 +76,13 @@ class RMFPProDaysSinceSubscribedMatchingAttribute @Inject constructor(
     ): MatchingAttribute? {
         return when (key) {
             PProDaysSinceSubscribedMatchingAttribute.KEY -> {
-                if (jsonMatchingAttribute.verifyValidValues()) {
+                try {
                     PProDaysSinceSubscribedMatchingAttribute(
                         min = jsonMatchingAttribute.min.toIntOrDefault(-1),
                         max = jsonMatchingAttribute.max.toIntOrDefault(-1),
                         value = jsonMatchingAttribute.value.toIntOrDefault(-1),
                     )
-                } else {
+                } catch (e: Exception) {
```

Comment:
If incorrect types are passed to the json attribute, it will throw an exception. If it happens, we don’t match it even if the key is correct.